clang/docs/ReleaseNotes.rst

```diff
@@ -586,6 +586,20 @@ Improvements to Clang's time-trace
 - Clang now specifies that using ``auto`` in a lambda parameter is a C++14 extension when
   appropriate. (`#46059: <https://github.com/llvm/llvm-project/issues/46059>`_).
 
+Improvements to Coverage mapping
+--------------------------------
+
+- Macros defined in system headers are not expanded in coverage
+  mapping. Conditional expressions in system header macros are no
+  longer taken into account for branch coverage. They can be included
+  with ``-mllvm -system-headers-coverage``. (#78920)
+- MC/DC coverage has been improved. (#82448)
+  - The maximum number of conditions is no longer limited to 6. See
+    `this <SourceBasedCodeCoverage.html#mc-dc-instrumentation>` for
+    more details.
+  - The format of profraw generated by the runtime is not backwards
```

Comment:
I don't understand this one. Was it backwards compatible before?

Reply:
It'd be appropriate to drop mentioning backward compatibility here, since it is our common sense.

I wanted to mention incompatibility of data semantics. Apparently, only `llvm-cov` would be affected.